source/TestAdapter/TestObjectHelper.cs

```diff
@@ -28,11 +28,9 @@ public static string GetNanoClrLocation()
             {
                 return nanoClrFullPath;
             }
-            else
-            {
-                var inititialDir = new DirectoryInfo(Path.GetDirectoryName(thisAssemblyDir));
-                return FindNanoClr(inititialDir);
-            }
+            
```

Comment:
Your change will have the method perform the same way, no questions about it! 😃 But (again!) I prefer to have these coded with the `else`. It greatly improves code readability as it shows the intent of the `else` block being executed as an alternative to the `if` clause. 
When you remove the `else`, you're "closing" the matter and "moving along", which makes that code block "loose" from the `if` from which it belongs to.